Library/Formula/puddletag.rb

```diff
@@ -41,6 +41,12 @@ def install
     end
 
     cp_r buildpath/"source/.", buildpath
+    inreplace buildpath/"puddlestuff/puddleobjects.py" do |s|
+      s.gsub! /QPalette.Window/, "pal.window()"
```

Comment:
Thanks. Please use upstream’s patch with a short comment saying why it’s necessary and that we can remove it at the next upgrade. Add this block above `def install`:

```rb
# Upstream commit to fix an issue with PyQT 4.11.4. Remove on next version.
patch do
  url "https://github.com/keithgg/puddletag/commit/489acd2ee62eb5fbff95f8220dc8958c14871931.patch"
  sha256 "fce0cfce4d4477cde4827a0a4d3ef74fbabf630ada2d0cf035cf155a17c37a68"
end
```